lib/msal-core/src/UserAgentApplication.ts

```diff
@@ -1723,7 +1737,7 @@ export class UserAgentApplication {
       const accessTokenCacheItems = this.cacheStorage.getAllAccessTokens(this.clientId, authority);
       for (let i = 0; i < accessTokenCacheItems.length; i++) {
         const accessTokenCacheItem = accessTokenCacheItems[i];
-        if (accessTokenCacheItem.key.userIdentifier === response.account.userIdentifier) {
+        if (accessTokenCacheItem.key.userIdentifier === response.getAccount().userIdentifier) {
```

Comment:
is account ever null ?

Reply:
saveAccessToken is called when an accessToken has been sent by the server, which would require an idToken and, subsequently, the account object. It is also called when the app saves the idToken as an access token for the application.

Reply:
👍